saifa/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  backend:
+    image: patranakamo/saifa-be:v1.3.3
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - "9988:80"
+    volumes:
+      - ${APP_DATA_DIR}/data/app:/data
```

Comment:
Can you please add this directory to this repo so that it is created for users when installing the app. If we don't do this then Docker will create the directory when the container is spun up, which has the potential to cause permissions issues.

See the Lightning Node app as an example:
https://github.com/getumbrel/umbrel-apps/tree/master/lightning

You'll want to create a `data` directory and then a `app` directory inside of that with a `.gitkeep` file to make sure the empty directory  is committed.